src/satosa/exception.py

```diff
@@ -30,6 +30,39 @@ class SATOSACriticalError(SATOSAError):
     """
     pass
 
+class SATOSAProcessingHaltError(SATOSAError):
+    """
+    SATOSA error that should stop processing
+    """
+    def __init__(self, state, message, *args, **kwargs):
+        """
+        :type state: satosa.state.State
+        :param message: str
+        :param args: any
+        :param kwargs: any
+
+        :param state: Satosa state
+        :param message: A test message
+        :param args: whatever
+        :param kwargs: whatever
+        """
+        super().__init__(message, *args, **kwargs)
+        self._message = "One of the plugins or microservices raised an error. Redirect to an error page"
```

Comment:
What is misleading ? This is raised either in a backend/frontend or in a microservice. What makes you think it's microservice specific ?